src/main/java/oo/lang/bool/TrueClass.java

```diff
@@ -0,0 +1,11 @@
+package oo.lang.bool;
+
+public class TrueClass extends Bool {
+    public static final TrueClass True = new TrueClass();
+
```

Comment:
We can create as many `True` values as we want. But it supposed to be a singleton class.

Reply:
same